docs/sdk/tutorials/label_parsing.md

```diff
@@ -514,7 +516,7 @@ print(
 
```

Comment:
I missed this one but I think that above, the `Standard object detection job` part should be renamed `Bounding Box detection job`